server/dash/dashboards/neighborhood.py

```diff
@@ -37,63 +39,90 @@ def populate_options():
     return values
 
 
+# looks good, need to clean the code and add also territory size as well as density
+
 # Define callback to update graph
 @app.callback(
     Output("graph1", "figure"),
     [Input("council_list", "value"), Input("data_type", "value")],
 )
 def update_figure(selected_council, selected_timeframe):
-    """Creates comparison graph
+    """Creates comparison graph.
 
     Args:
-        selected_council:
-          A string representing the council selected from the drop down menu.
-        selected_timeframe:
-          An integer representing the number of days around which to compute the moving average.
-          This is selected with radio buttons at the bottom of the graph.
+        selected_council: A string representing the council selected
+          from the drop down menu.
+        selected_timeframe: An integer representing the number of days around which to
+          compute the moving average. This is selected with radio buttons at
+          the bottom of the graph.
 
     Returns:
-        fig:
-          Plotly graph that compares the selected moving average between the chosen council
-          and the moving average of the avg of all the 99 neighborhood councils.
+        Plotly graph that compares the selected moving average between the
+          chosen council and the moving average of the avg of all the 99 neighborhood councils.
     """
+    NC_POP = int(
+        pop[pop.council_name == selected_council].sum_pop20
+    )  # population of selected council.
+
+    NC_MOVING_AVERAGE_KEY = (
+        "nc_ma"  # Key for neighborhood council moving average in dataframe (total/99).
+    )
+    NC_POP_MOVING_AVERAGE_KEY = "nc_ma_pop"  # Key for neighborhood council moving average adjusted to per 10,000 people.
+    SELECT_NC_MOVING_AVERAGE_KEY = "select_nc_ma"  # Key for selected neighborhood council moving average in dataframe.
+    SELECT_NC_POP_MOVING_AVERAGE_KEY = "select_nc_ma_pop"  # Key for selected neighborhood council moving average in dataframe adjusted to per 10,000 people.
```

Comment:
Please move these constants to the top of the file where other constants are defined.